tests/e2e/PolarChart/PolarChart.spec.ts

```diff
@@ -6,24 +6,25 @@ xdescribe("Polar chart", () => {
     });
 
     xit("should generate a chart", () => {
-        chart.polar.waitForVisible();
+        chart.polar.waitForDisplayed();
         const nodeName = chart.polar.getAttribute("nodeName");
 
         expect(nodeName).toBe("svg");
     });
 
     xit("should be generated with two traces", () => {
-        chart.traces.waitForValue();
-
-        expect(chart.traces.value.length).toBe(2);
+        browser.waitUntil(() => {
```

Comment:
Same things in this file

Reply:
This spec is skipped.

Reply:
I think it's better to fix it now, because the next time it's likely to be forgotten.